src/cross_module_fn.h

```diff
@@ -98,6 +98,7 @@ typedef struct CrossModuleFunctions
 	PGFunction compressed_data_out;
 	bool (*process_compress_table)(AlterTableCmd *cmd, Hypertable *ht,
 								   WithClauseResult *with_clause_options);
+	void (*process_altertable_cmd)(Hypertable *ht, const AlterTableCmd *cmd);
```

Comment:
I would try to avoid adding new functions here, for example we have `ddl_command_start` hook already which basically allows to handle process utility hook on the tsl side.

If you prefer this approach more, I suggest to make argument ordering consistent with the function below